build-system/babel-config/minified-config.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS-IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const argv = require('minimist')(process.argv.slice(2));
+const {getReplacePlugin} = require('./helpers');
+
+/**
+ * Gets the config for minified babel transforms run, used by 3p vendors.
+ *
+ * @return {!Object}
+ */
+function getMinifiedConfig() {
```

Comment:
It could, since the code runs within iframe, it is a completely separate environment than amp. I have removed several transformers that are not necessary.